test/test_distributions.py

```diff
@@ -82,15 +90,19 @@ def __new__(cls, jax_dist, *params):
     T(dist.HalfCauchy, np.array([1., 2.])),
     T(dist.HalfNormal, 1.),
     T(dist.HalfNormal, np.array([1., 2.])),
-    T(dist.LogNormal, 1., 0.2),
-    T(dist.LogNormal, -1., np.array([0.5, 1.3])),
-    T(dist.LogNormal, np.array([0.5, -0.7]), np.array([[0.1, 0.4], [0.5, 0.1]])),
     T(dist.LKJCholesky, 2, 0.5, "onion"),
     T(dist.LKJCholesky, 2, 0.5, "cvine"),
     T(dist.LKJCholesky, 5, np.array([0.5, 1., 2.]), "onion"),
     T(dist.LKJCholesky, 5, np.array([0.5, 1., 2.]), "cvine"),
     T(dist.LKJCholesky, 3, np.array([[3., 0.6], [0.2, 5.]]), "onion"),
     T(dist.LKJCholesky, 3, np.array([[3., 0.6], [0.2, 5.]]), "cvine"),
+    T(dist.LogNormal, 1., 0.2),
+    T(dist.LogNormal, -1., np.array([0.5, 1.3])),
+    T(dist.LogNormal, np.array([0.5, -0.7]), np.array([[0.1, 0.4], [0.5, 0.1]])),
+    T(dist.MultivariateNormal, 0., np.array([[1., 0.5], [0.5, 1.]]), None, None),
+    T(dist.MultivariateNormal, np.array([1., 3.]), None, np.array([[1., 0.5], [0.5, 1.]]), None),
```

Comment:
The addition of optional parameters makes these fixtures somewhat wonky - maybe it will be nice to support kwargs so that we do not need to filter for `None` in tests. Should be a separate PR though.

Reply:
Yeah, it might be more explicit if we define subclasses in the test:
```
def MultivariateNormalCov(dist.MultivariateNormal):
    def __init__(self, loc, cov):
        super(dist.MultivariateNormal, self).__init__(loc, covarariance_matrix=cov)
```
Do you want me to make that change?

Reply:
Hmm, I made an attempt but it is a bit complicated to handle `inspect` code. Will defer to another PR where we meet this `None` pattern again.

Reply:
Don't worry about it. This is already a big PR, and it might actually not be worth adding more complexity just for a single test case. :)

Reply:
I think the subclassing idea might be nice along with a wrapper function like our logit / prob parametrization. Let us worry about it in a different PR though.